packages/client/src/offline/reducer.ts

```diff
@@ -384,8 +412,18 @@ function reducer(
       }) as ICertificateTemplateData
 
       const certificatesTemplates = {
-        birth: { svgCode: birthCertificateTemplate.svgCode },
-        death: { svgCode: deathCertificateTemplate.svgCode }
+        birth: {
+          id: birthCertificateTemplate._id,
+          svgCode: birthCertificateTemplate.svgCode,
+          svgFileName: birthCertificateTemplate.svgFilename,
+          lastModifiedDate: birthCertificateTemplate.svgDateUpdated
+        },
+        death: {
+          id: deathCertificateTemplate._id,
+          svgCode: deathCertificateTemplate.svgCode,
+          svgFileName: deathCertificateTemplate.svgFilename,
+          lastModifiedDate: deathCertificateTemplate.svgDateUpdated
+        }
```

Comment:
```suggestion
        birth: {
          id: birthCertificateTemplate._id,
          definition: birthCertificateTemplate.svgCode,
          fileName: birthCertificateTemplate.svgFilename,
          lastModifiedDate: birthCertificateTemplate.svgDateUpdated
        },
        death: {
          id: deathCertificateTemplate._id,
          definition: deathCertificateTemplate.svgCode,
          fileName: deathCertificateTemplate.svgFilename,
          lastModifiedDate: deathCertificateTemplate.svgDateUpdated
        }
```
This way I don't think we'll need to forcefully typecast right?

Reply:
Fixed